src/PluginsManager/Microsoft.Performance.Toolkit.PluginsManager.Core.NuGet/NuGetPluginDiscoverer.cs

```diff
@@ -105,7 +111,8 @@ public sealed class NuGetPluginDiscoverer : IPluginDiscoverer
                         packageMetadata.Title,
                         packageMetadata.Description,
                         this.pluginSource.Uri,
-                        Guid.Parse(PluginManagerConstants.NuGet));
+                        FetcherResourceId,
```

Comment:
Bit confused here. Shouldn't the fetcher ID be taken from the metadata of the plugin that gets pulled?

Reply:
What metadata are you referring to? The metadata from the plugin package? I don't think the metadata should know what fetcher should be used. The discoverer should provide the fetcher id.

Reply:
This was in reference the FetcherResourceId. I see there's the TODO